lib/src/solver/report.dart

```diff
@@ -130,6 +130,40 @@ class SolveReport {
     }
   }
 
+  /// Displays a two-line message, number of outdated packages and an
+  /// instruction to run `pub outdated` if outdated packages are detected.
+  void reportOutdated() {
+    var outdatedPackagesCount = 0;
+
+    var names = _result.packages.map((id) => id.name).toList();
```

Comment:
No need to convert with `.toList()`.

Also avoid `var` when the variable is never reassigned, you can just use `final`.

Reply:
Done.